docs/api/server/Environment-Variables.md

```diff
@@ -421,11 +422,41 @@ ONE_CLIENT_REPORTING_URL=https://my-app-errors.com/client
 **Default Value**
 ```bash
 # if NODE_ENV=development
-ONE_CLIENT_REPORTING_URL=/_
+ONE_CLIENT_REPORTING_URL=/_/report/errors
```

Comment:
If you're going to do this, may as well change the env var name altogether.
```suggestion
ONE_CLIENT_ERROR_REPORTING_URL=/_/report/errors
```

Reply:
Won't this also require a change to one-app-ducks?

Reply:
changing the name would require a change which is why i left it, currently one-app-ducks is treating this as `/_/report/errors` - https://github.com/americanexpress/one-app-ducks/blob/master/src/errorReporting.js#L109

Reply:
carry on 🖖